app/controllers/claims_controller.rb

```diff
@@ -3,8 +3,31 @@ def new
   end
 
   def create
-    TslrClaim.create!
+    claim = TslrClaim.create!
+    session[:tslr_claim_id] = claim.to_param
 
     redirect_to qts_year_claim_path
   end
+
+  def update
+    current_claim.update_attributes(claim_params)
+    redirect_to claim_school_claim_path
+  end
+
+  def qts_year
+  end
+
+  def claim_school
+  end
+
+  private
+
+  def claim_params
+    params.require(:tslr_claim).permit(:qts_award_year)
+  end
+
+  def current_claim
+    @current_claim ||= TslrClaim.find(session[:tslr_claim_id])
+  end
+  helper_method :current_claim
```

Comment:
Is this a prefered style for the place to declare these `helper_methods` I see them alot at the top of the file?

Reply:
It's not a rigid convention, but generally folks place them together somewhere near the top off the controller code. Personally I do like having them near the method definition but I'm not wedded to it.